js/tests/components/song/list.spec.js

```diff
@@ -136,4 +138,128 @@ describe('components/song/list', () => {
     wrapper.find('.song-list-wrap').trigger('keydown.a', { ctrlKey: true })
     wrapper.vm.filteredItems.forEach(item => expect(item.selected).toBe(true))
   })
+
+  describe(`mounted()`, () => {
+    let orderBy
+    beforeEach(() => {
+      orderBy = jest.spyOn(filters, "orderBy")
+    })
+
+    it(`when sorting is disallowed, it will not sort`, () => {
+      shallowMount(Component, {
+        propsData: {
+          items: songs,
+          type: "queue",
+          sortable: false
+        }
+      })
+
+      expect(orderBy).not.toHaveBeenCalled()
+    })
+
+    it(`will not sort songs by default`, () => {
+      shallowMount(Component, {
+        propsData: {
+          items: songs,
+          type: "all-songs"
+        }
+      })
+
+      expect(orderBy).toHaveBeenCalledWith(expect.any(Array), "", 1)
+    })
+
+    it(`when it's an album's song list,
```

Comment:
This doesn't read as a sentence to me. How about something like this?

```suggestion
    it('sorts songs by track and disc number in album song list')
```